### PR TITLE
AppConfig now gets DB properties from file

### PR DESCRIPTION
**Pull request by Borisko130**

### CODE REVIEW

**Review by vladkuzya — APPROVED** [review]
Good)

File: src/main/java/spring/intro/config/AppConfig.java [hunk 1]
```diff
@@ -3,25 +3,32 @@
...
 
 @Configuration
+@PropertySource("classpath:db.properties")
 @ComponentScan(basePackages = {
         "spring.intro.service",
         "spring.intro.dao"
 })
 public class AppConfig {
+    @Autowired
+    private Environment environment;
```
Comment (RomanDubovskyi) [inline src/main/java/spring/intro/config/AppConfig.java:23]: let's use constructor injection

**Review by RomanDubovskyi — COMMENTED** [review]
change PR name based on functional, you've implemented

**Comment by RomanDubovskyi** [comment]
travis failed

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**